tests/test_dpo_trainer.py

```diff
@@ -161,7 +167,9 @@ def test_dpo_trainer_without_providing_ref_model(self):
 
     @require_peft
     @mark.peft_test
-    def test_dpo_trainer_without_providing_ref_model_with_lora(self):
+    @parameterized.expand([["gpt2", True], ["t5", False]])
```

Comment:
Similar question here as above:
```suggestion
    @parameterized.expand([["gpt2", True], ["t5", True]])
```